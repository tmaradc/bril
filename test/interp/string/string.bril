@main() {
  s: string = const "Alo Mundo";
  end: string = const "!!!";
  s2: string = strcat s end;
  print s;


  tam1: int = strlen s;
  tam2: int = strlen end;
  tam3: int = strlen s2;
  print tam1 tam2 tam3;

  c: char = strchar s2 tam1;
  print c;
}