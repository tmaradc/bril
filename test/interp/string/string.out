Alo Mundo
9 3 12
!
